.scripts/build_docs.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env sh
+
+# abort on errors
+set -e
+
+OUTPUT_DIR=docs/docs_page
+TMP=docs/tmp
+
+#clean and copy images
+if [ -d "$OUTPUT_DIR" ]; then rm -Rf $OUTPUT_DIR; fi
```

Comment:
No need to check if the directory exists

```suggestion
rm -Rf $OUTPUT_DIR
```

Reply:
See b81117c